dashboard/app/views/home/_homepage.html.haml

```diff
@@ -4,7 +4,7 @@
 - homepage_data[:userid] = current_user.id
 - homepage_data[:valid_grades] = Section.valid_grades
 - homepage_data[:topCourse] = @top_course
-- homepage_data[:isenglish] = @is_english
+- homepage_data[:isEnglish] = @is_english
```

Comment:
it would be good to settle on a casing for these parameters but this one seems consistent with at least one of the above :)